arex-instrumentation-api/src/main/java/io/arex/inst/extension/matcher/IgnoreClassloaderMatcher.java

```diff
@@ -1,12 +1,22 @@
 package io.arex.inst.extension.matcher;
 
+import io.arex.agent.bootstrap.util.CollectionUtil;
 import io.arex.agent.bootstrap.util.StringUtil;
+import java.util.List;
 import net.bytebuddy.matcher.ElementMatcher;
+import javax.annotation.Nonnull;
 
 public class IgnoreClassloaderMatcher extends ElementMatcher.Junction.AbstractBase<ClassLoader> {
 
     static final String BYTE_BUDDY_PREFIX = StringUtil.removeShadePrefix("net.bytebuddy.");
-    private final ElementMatcher<ClassLoader> matcher;
+
+    private static final List<String> IGNORED_CLASSLOADER_PREFIXES = CollectionUtil.newArrayList(
+        "sun.reflect.", "jdk.internal.reflect.", IgnoreClassloaderMatcher.BYTE_BUDDY_PREFIX);
+    private ElementMatcher<ClassLoader> matcher;
+
+    public IgnoreClassloaderMatcher(@Nonnull List<String> ignoreClassLoaderPrefixes) {
```

Comment:
NR: not recommended to use the annotation `@Nonnull`, AREX agent will not package this class into the jar.